wiki/en/Running-a-Server.md

```diff
@@ -298,6 +297,49 @@ You yourself should connect using the local network (LAN) address of the machine
 
 Most domestic internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Please research how to do that for your specic set up. Your router might support some "dynamic DNS" providers out of the box. If this is not the case, set up a dynamic DNS client as described by the dynamic DNS provider you chose.
```

Comment:
```suggestion
Most residential internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Please research how to do that for your specic set up. Your router might support some "dynamic DNS" providers out of the box. If this is not the case, set up a dynamic DNS client as described by the dynamic DNS provider you chose.
```

Reply:
Is this what's meant by _domestic_?

Reply:
"Residential" is probably the better term, since "domestic" can also imply the opposite of "international".